v2/spiffebundle/bundle.go

```diff
@@ -0,0 +1,43 @@
+package spiffebundle
+
+import (
+	"crypto/x509"
+	"io"
+
+	"gopkg.in/square/go-jose.v2"
+)
+
+// Bundle is a SPIFFE bundle
+type Bundle struct {
+	jose.JSONWebKeySet
+
+	Sequence    uint64 `json:“spiffe_sequence,omitempty”`
```

Comment:
wrogn character `“` you must replace them with `"`

Reply:
wow.